docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+```
+3.	Create a JSON configuration file with the following schema:
+```sh
+vim continuouspatching.json //any file creation command and file name will do
+```
+Here is the JSON Schema:
+```sh
+{
+    "version": "v1",
+    "tag-convention" : "<incremental|floating>",
+    "repositories": [{
+        "repository": "<Repository Name>",
+        "tags": ["<comma-separated-tags>"],   
+        "enabled": <true|false>
+    }] 
+}
+```
+The schema ingests specific repositories and tags in an array format. Each variable is defined below:
+
+- "version" allows the ACR team to track what schema version you’re on. Do not change this variable unless instructed to.
+- "tag-convention" this is an optional field. Allowed values are "incremental" or "floating" (defaults to incremental if not specified).
+  - incremental = appends a structured -nnn suffix to the original tag, where -nnn increments from 1 to 999. The highest existing -nnn suffix will always be the one incremented for subsequent patches
+  - floating = creates a single mutable tag formatted as "-patched" that is appended to the original tag and always points to the latest patched image
+
+    We recommend using incremental tags as a more secure method of updating patches for deployments. Incremental tags are immutable, which aligns with cloud native best practices (easy rollbacks).
+
+    As for floating tags, they can be used conveniently for builds at build time, since the consistent tag always points to the latest patch and allows teams to avoid the constant manual switch to new patched tags.
+  
+    Here's an example:
+  
+    ![PatchingTimelineExample](./media/patching_timeline_example1.png)
+
+- "repositories" is an array that consists of all objects that detail the specific repository and tags
+    - "repository" refers to repository name
+    - "tags" is an array of tags separated by commas. The wildcard "*" can be used to signify all tags within that repository
+    - "enabled" is a Boolean value of true or false determining if the specified repo is on or off
+
+The following details an example configuration for a customer who wants to patch all tags(use the * symbol) within the repository “python”, and to patch specifically the “jammy-20240111” and “jammy-20240125” tags in the repository “ubuntu”. 
+
+JSON example:
+```json
+{
+"version": "v1",
+"tag-convention" : "incremental",
+"repositories": [{
+        "repository": "python",
+        "tags": ["*"],
+        "enabled": true
+    },
+    {
+        "repository": "ubuntu",
+        "tags": ["jammy-20240111", "jammy-20240125"],
+        "enabled": true, 
+    }]
+}
+```
+4. After creating your configuration file, it is recommended to execute a dry run to verify the intended artifacts are selected by the JSON criteria. The dry run requires a parameter called schedule, which specifies how often your continuous patching cycle will run. The schedule flag is measured in days, with a minimum value of 1 day, and a maximum value of 30 days. 
```

Comment:
Perhaps add an example here highlighting that the `--schedule` parameter decides the frequency, and that it will run indefitnitely.